frontend/src/app/business-plans/list/layout.tsx

```diff
@@ -91,18 +93,22 @@ function BusinessPlansList(props: PropsWithChildren) {
 }
 
 function BPListHeader() {
+  const { isViewer } = useContext(BPContext) as any
+
   return (
     <div className="mb-8 flex items-center justify-between">
       <PageHeading>Business Plans</PageHeading>
-      <CustomLink
-        className="px-4 py-2 text-lg uppercase"
-        color="secondary"
-        href="/business-plans/create"
-        variant="contained"
-        button
-      >
-        Create new plan
-      </CustomLink>
+      {!isViewer && (
```

Comment:
@dana-cfc4 Sorry for commenting so late, but maybe we could instead have a `const userCanEditBusinessPlan` defined in `frontend/src/@types/user_types.ts` so we can start to slowly move away from user type checks in the frontend components code.